l10n_it_ricevute_bancarie/README.rst

```diff
@@ -113,6 +113,7 @@ Contributors
 * Alex Comba <alex.comba@agilebg.com>
 * Marco Calcagni <mcalcagni@dinamicheaziendali.it>
 * Sergio Zanchetta <https://github.com/primes2h>
+* Simone Vanin <simone.vanin@agilebg.com>
```

Comment:
Invece di modificare il README direttamente, dovresti modificare i file dentro la cartella `readme` perché questo file verrà rigenerato durante il merge (vedi https://github.com/OCA/oca-github-bot/#on-push-to-main-branches, che userà quindi https://github.com/OCA/maintainer-tools#readme-generator) e i tuoi cambiamenti andranno persi.